src/main/java/org/fundacionjala/coding/wilson/EanValidation.java

```diff
@@ -0,0 +1,40 @@
+package org.fundacionjala.coding.wilson;
+
+/**
+ *
+ */
+public final class EanValidation {
+
+    /**
+     *
+     */
+    private EanValidation() {
+    }
+
+    /**
+     *
+     * @param eanCode is a code
+     * @return suma control
+     */
+    public static boolean validate(final String eanCode) {
+        final char base = '0';
+        final int limited = 12;
+        final int base2 = 10;
+
+        final int impar = 1;
+        final int par = 3;
+
+        int sum = 0;
+        int aux = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "aux" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)